hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/package-info.java

```diff
@@ -72,12 +72,25 @@
  * hashtables (hence the name). It consumes the enriching streams in full
  * before ingesting any data from the primary stream.
  * <p>
- * The output is just like an SQL left-outer join: for each primary item
- * there are N output items, one for each matching item in the enriching
- * set. If an enriching set doesn't have a matching item, the output will
- * have a {@code null} instead of the enriching item. The join also allows
+ * The output of {@code hashJoin} is just like an SQL left-outer join:
```

Comment:
Please reflow these paragraphs.

Reply:
> 
> 
> Please reflow these paragraphs.

By reflow you mean only formatting or also how sentences are written?

Reply:
Just the formatting aspect :)